File: travel_restrictions/v2/get/Java SDK/TravelRestrictions.java
// How to install the library at https://github.com/amadeus4dev/amadeus-java

import com.amadeus.Amadeus;
import com.amadeus.Params;
import com.amadeus.exceptions.ResponseException;
import com.amadeus.resources.DiseaseReport;

public class TravelRestrictions {

  public static void main(String[] args) throws ResponseException {

    Amadeus amadeus = Amadeus
        .builder("YOUR_AMADEUS_API_KEY","YOUR_AMADEUS_API_SECRET")
        .build();

    //What are the Covid19 restrictions for the USA?
    DiseaseReport diseaseReport = amadeus.dutyOfCare.diseases.covid19Report.get(Params.with("countryCode","US"));

    if (diseaseReport.getResponse().getStatusCode() != 200) {
        System.out.println("Wrong status code: " + diseaseReport.getResponse().getStatusCode());
        System.exit(-1);
    }

    System.out.println(diseaseReport);
  }
}

File: travel_restrictions/v2/get/Python SDK/travel_restrictions.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    What are the Covid19 restrictions for the USA?
    '''
    response = amadeus.duty_of_care.diseases.covid19_report.get(countryCode='US')
    print(response.data)
except ResponseError as error:
    raise error
